### ISSUE TITLE
Specifying triple redundancy mode elect 5 coordinators

### ISSUE THREAD

**Issue by iyuroch** [FoundationDB/fdb-kubernetes-operator#1819]
### What happened?

When I specify redundancy mode triple for coordinator fdb operator will elect 5 (cluster.MinimumFaultDomains() + cluster.DesiredFaultTolerance()).
It also force us to have more pods for coordinator (we have separate pods for coordinator to support physical backup).

### What did you expect to happen?

To have 3 coordinators

### How can we reproduce it (as minimally and precisely as possible)?

Deploy any cluster with redundancy mode triple

### Anything else we need to know?

I will be happy to contribute to fix the issue.

### FDB Kubernetes operator

<details>

```console
$ kubectl fdb version
# paste output here
```

</details>

### Kubernetes version

<details>

```console
$ kubectl version
# paste output here
```

</details>

### Cloud provider

<details>

</details>

**Comment by johscheuer** [comment]
I changed the label to enhancement as this is not a bug. The triple configuration is used for the transaction system and storage subsystem and will allow the failure of processes in 2 zones (as the data is replicated across 3 zones). Since the coordinators use Paxos we need 5 coordinators to tolerate the failure of 2 zones.

We can allow to modify the number of coordinators to choose if we think this makes sense. In your new desired configuration you could only lose 1 coordinator. In general it's a good practice to have at least 5 coordinators (maybe you can reduce the resources for the Pod if that's your concern as the coordinators are not in the commit/read critical path). But if you loose a majority of coordinators and you are not able to recover them, you have to restore the cluster from a backup.

**Comment by iyuroch** [comment]
Makes sense since it's paxos, thanks for explanation. I'm not sure if it makes sense to have additional configuration (since triple redundancy - means losing 2 coordinators).

**Comment by johscheuer** [comment]
Do you think this section in the docs explain that well enough: https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/docs/manual/fault_domains.md#coordinators or should we polish it?

**Comment by iyuroch** [comment]
I would include information about it being paxos thus reason for having that many
